### ISSUE TITLE
Wrong input of CANNET

### ISSUE THREAD

**Issue by CnChengGong** [mingcv/Bread#13]
https://github.com/mingcv/Bread/blob/e0c50680f38ac9f417ef6a233448f94e68f332f2/train_CAN.py#L95-L99
Should be
```python
    colors = self.model_canet(torch.cat([texture_in, cb_in, cr_in, texture_en, cb_en, cr_en],

**Comment by CnChengGong** [comment]
And 
https://github.com/mingcv/Bread/blob/e0c50680f38ac9f417ef6a233448f94e68f332f2/train_CAN.py#L106
should be
``` python
 image_out = kornia.color.ycbcr_to_rgb(torch.cat([texture_en, cb, cr], dim=1))

**Comment by CnChengGong** [comment]
And
https://github.com/mingcv/Bread/blob/e0c50680f38ac9f417ef6a233448f94e68f332f2/train_CAN.py#L106
Should be
```python
    image_out = kornia.color.ycbcr_to_rgb(torch.cat([texture_en, cb, cr], dim=1))

**Comment by mingcv** [comment]
Actually, the use of `texture_gt` here is a deliberate design choice intended to **decouple texture and color**.

The logic behind this implementation is: *"Assuming the texture is already perfectly aligned with the Ground Truth, how should the color be adjusted?"*

If we switch this input to `texture_en` (the enhanced texture), the CAN would be forced to compensate for texture artifacts generated by the previous stage. This entangles the two tasks and hinders the effective decomposition of sub-problems.

While using `texture_en` might yield performance gains on paired datasets like LOL (by overfitting to the specific pipeline's noise), it generally hurts the model's ability to **generalize to unseen data**. Therefore, strictly separating the inputs ensures the CAN focuses solely on color correction.
